upgrade/upd-2.4.0-to-2.4.1/index.php

```
@@ -235,12 +295,11 @@ public function xoops_stripeKey($xoops_key, $num = 6, $length = 30, $uu = 0)
         return $ret;
     }
 
-    public function __construct()
+    public function __construct(XoopsMySQLDatabase $db, UpgradeControl $control)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)